api/src/main/java/org/infinispan/api/Infinispan.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.api;
+
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.api.reactive.KeyValueStore;
+import org.infinispan.api.reactive.KeyValueStoreConfig;
+
+/**
+ * Each infinispan is a node instance. It can be embedded or client node, depending on the access point. {@link
```

Comment:
The use of `node` is confusing here. I'd normally associate `node` with a server-side node. I'd simply say this in an Infinispan instance, either embedded or a client.